### PR TITLE
Add initial addon list item styling; misc related

### PR DESCRIPTION
**Pull request by samhh**

### CODE REVIEW

**Review by sallaben — DISMISSED** [review]
Looks good to me. I added some styling. I also changed the format of the rating to a decimal between 0.00 and 1.00. We should discuss addons in more detail but for now I think a %-based rating would be more effective than a 1-5 star rating. Let me know what you think.

File: client/components/addons/list-item/addon-list-item.js [hunk 1]
```diff
@@ -8,7 +8,12 @@ const AddonListItem = ({ name, description, author, totalUsers, rating }) => (
 		key={name}
 		className={`c-card ${styles['item']}`}
 	>
-		<strong>{name}</strong> by {author}. Installs: {totalUsers}. Rating: {rating}.
+		<span className={`${styles['addon-title']}`}>
```
Comment (samhh) [inline client/components/addons/list-item/addon-list-item.js]: No need for template strings.

File: client/components/addons/list-item/addon-list-item.js [hunk 2]
```diff
@@ -8,7 +8,12 @@ const AddonListItem = ({ name, description, author, totalUsers, rating }) => (
...
+			<strong>{name}</strong> by {author}
+		</span>
+		<span className={`${styles['addon-details']}`}>
```
Comment (samhh) [inline client/components/addons/list-item/addon-list-item.js]: Here too.

**Review by samhh — COMMENTED** [review]
Only minor syntax changes needed. Note I only used template strings above when I wanted to combine classes. :-)

**Review by sallaben — APPROVED** [review]

**Pull request merged**